.github/workflows/build.yml

```diff
@@ -174,7 +174,7 @@ jobs:
       -
         name: GitHub Release
         if: startsWith(github.ref, 'refs/tags/v')
-        uses: softprops/action-gh-release@de2c0eb89ae2a093876385947365aca7b0e5f844  # v0.1.15
+        uses: softprops/action-gh-release@d99959edae48b5ffffd7b00da66dcdb0a33a52ee  # v0.1.15
```

Comment:
Looks like it doesn't update the comment 🤔 

@crazy-max any idea if dependabot can do that?

Reply:
Might not handle major properly, seems to work just fine https://github.com/tonistiigi/binfmt/pull/178

Reply:
Pushed extra commit to align tag and bump to latest stable as well.